src/index.test.js

```diff
@@ -87,6 +87,20 @@ describe("Router children", () => {
       )
     });
   });
+  it.only("allows for fragments", () => {
```

Comment:
gotta get rid of the `only`

Reply:
![so embarassed](https://media2.giphy.com/media/lwHA9RyGZB6Hm/giphy.gif?cid=8fc3c89720b16e69c43ef322b67c0d1c547ea9cd4d7830c4&rid=giphy.gif)